src/Previews.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import { getPreviews } from './queueserver'
+
+interface TabPanelProps {
+  children?: React.ReactNode;
+  index: any;
+  value: any;
+}
+
+function TabPanel(props: TabPanelProps) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`scrollable-auto-tabpanel-${index}`}
+      aria-labelledby={`scrollable-auto-tab-${index}`}
+      {...other}
+    >
+      {value === index && (
+        <Box p={3}>
+          <Typography>{children}</Typography>
+        </Box>
+      )}
+    </div>
+  );
+}
+
+type PreviewsProps = {
+  runUid: string;
+}
+  
+type PreviewsState = {
+  value: number;
```

Comment:
What is supposed to be stored in `value`?

Reply:
`value` stores the selected tab. The name `value` came from the material ui example code.

Reply:
Could you please make a comment about it next to the `value`?